aptos-move/aptos-release-builder/src/components/feature_flags.rs

```diff
@@ -69,6 +69,11 @@ pub fn generate_feature_upgrade_proposal(
 
     let writer = CodeWriter::new(Loc::default());
 
+    emitln!(writer, "// Modifying on-chain feature flags: ");
```

Comment:
I forgot, but did we add a check to not enable/disable features that are already enabled/disabled on chain?

Reply:
Right now the logic is: if any on chain config has modified we will send everything.